src/__tests__/Home.test.js

```diff
@@ -0,0 +1,10 @@
+import renderer from 'react-test-renderer';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Home from '../components/Logo';
+
+it('renders correctly', () => {
+  const home = renderer
+    .create(<Router><Home /></Router>)
+    .toJSON();
+  expect(home).toMatchSnapshot();
```

Comment:
- [ ] Great job on making testing on Home copmonent I think it woukd be great if you also test your details page since in the requirements to test all your components tested well 🔥💯✅🚀⭐⭐